### PR TITLE
Certified design fixes

### PR DESCRIPTION
**Pull request by carkod**
## Done

Search results and category search results filters have applied the following changes:
- Adapted filter titles to what is in the Accordion pattern originally (or Navigation).
- "Apply" and "Clear" filter buttons positioning

Not done:
- The section titles have a hover state when closed but not when open - this is a Vanilla discussion as there is discrepancy among designers, so [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/3950) was created.
- Remove left padding from filter buttons - I believe there is a `u-no-padding-right` already.

## QA

- check https://ubuntu-com-10224.demos.haus/certified/desktops (or any other item on the top navigation, the code comes from the same file)
- See that the accordion titles fit the [Vanilla accordion pattern](https://vanillaframework.io/docs/patterns/accordion) 
- Check that Apply and Clear filter buttons are aligned
- **Design should also review** the new strips for /desktops, /laptops, /devices ... (navigation on top) against [design](https://app.zeplin.io/project/60c1ca887b99f18a380e57e9/screen/60f169f46ff9dc0dccf77cc0)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4315

### CODE REVIEW

File: templates/certified/shared/category-search-results.html [hunk 1]
```diff
@@ -113,8 +111,8 @@ <h2 class="p-heading--4">
                 </li>
               </ul>
             </aside>
-            <button class="p-button--positive p-update-results js-apply-filters" type="submit" disabled>Apply filters</button>
-            <button class="p-button--neutral p-update-results" onclick="return clearFilters()">Clear filters</button>
+            <button class="p-button--neutral p-update-results" onclick="return clearFilters()">Clear</button>
+            <button class="p-button--positive p-update-results js-apply-filters" type="submit" disabled>Apply</button>
```
Comment (carkod) [inline templates/certified/shared/category-search-results.html:115]: This is just an order flip

**Comment by bethcollins92** [comment]
@spencerbygraves is this definitely ok with the filter applies and the count under the title? 
Screenshot 2021-09-02 at 11 55 37

**Review by bethcollins92 — APPROVED** [review]
Code and QA LGTM but just wanted to check the filter counter being on the second line as asked me to keep it at the top on a previous PR

**Comment by carkod** [comment]
> Code and QA LGTM but just wanted to check the filter counter being on the second line as @spencerbygraves asked me to keep it at the top on a previous PR

Ohh, that's probably not ok. Do you want to take over this PR? if not I'll see if I can do it.

**Comment by bethcollins92** [comment]
Yeah no probs @carkod I can take over it

**Comment by bethcollins92** [comment]
Fixed @spencerbygraves could you take another look please?

**Comment by bethcollins92** [comment]
Hi @long-chung - apologies I'm aware this is very out of context for you, but as Spencer is off we have no-one to design review this PR. Would you be able to take a look please? You should be able to follow the QA steps but please let me know if you're not sure

**Comment by long-chung** [comment]
@bethcollins92 Hiya, couple of things from me.
The labels of the buttons don't match up with the design was this discussed in a different post?
Buttons are inline as opposed to stacked, again not sure what the conversation around this was.
If the plan is to keep them inline, spacing needs to be reduced to 16px (2 rem)
Long label names on checkboxes should be inline with the text not the checkbox, or is that a Vanilla bug? (screenshot attached)
Screenshot 2021-09-06 at 10 35 42


Thanks

**Comment by carkod** [comment]
> @bethcollins92 Hiya, couple of things from me.
> The labels of the buttons don't match up with the design was this discussed in a different post?
> Buttons are inline as opposed to stacked, again not sure what the conversation around this was.
> If the plan is to keep them inline, spacing needs to be reduced to 16px (2 rem)
> Long label names on checkboxes should be inline with the text not the checkbox, or is that a Vanilla bug? (screenshot attached)
> Screenshot 2021-09-06 at 10 35 42
> 
> Thanks

Last item, I am doing it on Vanilla [**right now**](https://github.com/canonical-web-and-design/vanilla-framework/pull/3965)

**Comment by bethcollins92** [comment]
Hi @long-chung - thanks a lot for the review. Yes it was requested by Lyubo  who requested the changes with the buttons [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10125#issuecomment-893533632) - I've fixed them so they're right aligned. Carlos is working on the final point on vanilla as we speak so this should be fixed very soon

**Comment by bethcollins92** [comment]
@long-chung if you're happy with it (apart from the vanilla issue) would you be able to give it a design +1 please? once the vanilla bug is fixed the text should line up.

**Comment by long-chung** [comment]
@bethcollins92 Thanks for sorting. 
Sorry to be a pain; I think the CTAs need a divider line like you specced [here](https://github.com/canonical-web-and-design/ubuntu.com/pull/10125#issuecomment-893533632) as it feels like they are just floating.
This isn't a blocker so i'll +1 it. 

Thanks.

**Comment by bethcollins92** [comment]
ahh yes apologies I missed that. sure I'll add one now - thank you!

**Pull request merged**